llm-examples/mpt/mpt-7b/MLFlow Logging and inference with MPT-7B model.py

```diff
@@ -0,0 +1,155 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Load and Inference MPT-7B model on Databricks
+# MAGIC
+# MAGIC Environment for this notebook:
+# MAGIC - Runtime: 13.2 GPU ML Runtime
+# MAGIC - Instance: `g5.4xlarge` on AWS
+
+# COMMAND ----------
+
+# MAGIC %pip install xformers einops flash-attn==v1.0.3.post0 triton==2.0.0.dev20221202
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ### Log the model to MLFlow
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC Define a customized PythonModel to log into MLFlow.
+
+# COMMAND ----------
+
+import pandas as pd
+import numpy as np
+import transformers
+import mlflow
+import torch
+
+class MPT(mlflow.pyfunc.PythonModel):
+    def load_context(self, context):
+        """
+        This method initializes the tokenizer and language model
+        using the specified model repository.
+        """
+        # Initialize tokenizer and language model
+        self.tokenizer = transformers.AutoTokenizer.from_pretrained(
+          context.artifacts['repository'], padding_side="left")
+
+        config = transformers.AutoConfig.from_pretrained(
+            context.artifacts['repository'], 
+            trust_remote_code=True
+        )
+        # support for flast-attn and openai-triton is coming soon
+        # config.attn_config['attn_impl'] = 'triton'
```

Comment:
If `config.attn_config['attn_impl'] = 'triton'` is not supported yet, shall we keep it? Or is it actually supported now?
Wondering whether this support refers to MLR to MLflow.

Reply:
The support is mainly for model serving part.